src/components/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../styles/Button.css';
+
+function Button({ text, className, fun }) {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const Button = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/ernestmusong/Booking-app-frontend/assets/57421823/f7357d09-e4f7-402f-a2b2-42b51e01939b)


That way, your code will meet the latest standards. 💪

Reply:
hi @Meltrust  
Thanks for the suggestions we are refactoring it